using CoreclrTestLib;
using Xunit;

namespace baseservices_exceptions
{
    public class _generics_nested_try_catch05_nested_try_catch05_
    {
        [OuterLoop]
        [Fact]
        public void _generics_nested_try_catch05_nested_try_catch05_cmd()
        {
            CoreclrTestWrapperLib.RunTest("baseservices\\exceptions\\generics\\nested-try-catch05\\nested-try-catch05.cmd");
        }
    }
}
